unstructured_inference/inference/layoutelement.py

```diff
@@ -108,7 +108,11 @@ def merge_inferred_layout_with_extracted_layout(
                 continue
         region_matched = False
         for inferred_region in inferred_layout:
-            if inferred_region.source in (Source.CHIPPER, Source.CHIPPERV1):
+            if inferred_region.source in (
+                Source.CHIPPER,
+                Source.CHIPPERV1,
```

Comment:
We might want to include `Source.CHIPPERV2` as well, if the intent here is "any chipper source". In fact we might want a constant that includes all chipper sources.

Reply:
Added a constant. Thanks!